test/namespace.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// namespace.go provides methods to create and delete namespaces.
+
+package test
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
```

Comment:
Is this warranted? Or we see need in re-usable blocks like this for future tests?
Especially delete looks redundant.

Reply:
It's not necessary for this test specifically. It could be used for future tests if there's a need to, but I suppose it can be merged into the test itself for now.

Reply:
I am fine either way, just wanted to make sure we think about this :)

Reply:
Since it doesn't seem like you feel strongly about this, I've reverted to putting this back in as part of the changes.